pkg/apis/pipeline/v1beta1/pipeline_validation_test.go

```diff
@@ -3507,6 +3542,40 @@ func Test_validateMatrix(t *testing.T) {
 					Name: "barfoo", Value: ParamValue{Type: ParamTypeArray, ArrayVal: []string{"bar", "foo"}},
 				}}},
 		}},
+	}, {
+		name: "parameters in include matrix are strings",
+		tasks: PipelineTaskList{{
+			Name:    "a-task",
+			TaskRef: &TaskRef{Name: "a-task"},
+			Matrix: &Matrix{
+				Include: []MatrixInclude{{
+					Name: "test",
+					Params: []Param{{
+						Name: "foobar", Value: ParamValue{Type: ParamTypeString, StringVal: "foo"},
+					}, {
+						Name: "barfoo", Value: ParamValue{Type: ParamTypeString, StringVal: "bar"}},
+					}}},
+			},
+		}},
+	}, {
+		name: "parameters in include matrix are arrays",
```

Comment:
this test is missing from `v1/pipeline_validation_test.go`, should we include it in there too?

Reply:
Thanks for the catch! This has been added.